cagent.go

```diff
@@ -59,14 +59,15 @@ func New(cfg *Config, version string) *Cagent {
 
 	ca.SetLogLevel(ca.Config.LogLevel)
 
+	return ca
+}
+
+func (ca *Cagent) Initialize() error {
```

Comment:
Is there a reason why we can't have `Initialize` `New`?

Reply:
done